tests/bootstrap.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Predis-Toolkit
+ *
+ * Additional classes and functionality to extend Predis
+ *
+ * @copyright 2015 One Mighty Roar
+ * @link http://onemightyroar.com
+ */
+
+// Load our autoloader, and add our Test class namespace
+$autoloader = require(__DIR__ . '/../vendor/autoload.php');
+$autoloader->add('OneMightyRoar\PredisToolki\Tests', __DIR__);
```

Comment:
haha, not sure
probs because of PHPUnit's "load everything in the target directory" YOLO style?